src/dash-table/components/Table/props.ts

```diff
@@ -101,6 +119,27 @@ export interface IPaginationSettings {
     page_size: number;
 }
 
+export interface IUserInterfaceCell {
+    height: number;
+}
+
+export interface IUserInterfaceViewport {
+    scrollLeft: number;
+    scrollTop: number;
+    height: number;
+    width: number;
+}
+
+export interface IState {
+    forcedResizeOnly: boolean;
+    scrollbarWidth: number;
+    uiViewport?: IUserInterfaceViewport;
+    uiCell?: IUserInterfaceCell;
+    uiHeaders?: IUserInterfaceCell[];
+}
+
+export type StandaloneState = IState & Partial<PropsWithDefaultsAndDerived>;
```

Comment:
The table's standalone state is the combination of the connected props + basic state props